ONSdigital/sdx-downstream
# Standards changes

**Changes**

- Add flake/pep8 checks
- Use structured logging
- Update readme
- Use an asynchronous consumer
- Add unit tests

**How to test**

Run the unit tests. Run as part of compose setup and check messages are correctly delivered.

**Who can test**

Anyone but @iwootten
